apps/etterlatte-saksbehandling-ui/.nais/dev.yaml

```diff
@@ -56,6 +56,10 @@ spec:
       value: http://etterlatte-behandling
     - name: VEDTAK_API_URL
       value: http://etterlatte-vedtaksvurdering
+    - name: AXSYS_API_URL
+      value: https://axsys.dev-fss-pub.nais.io
+    - name: AXSYS_API_SCOPE
+      value: api://dev-fss.org.axsys/.default
   accessPolicy:
     outbound:
       external:
```

Comment:
Tror du må legge til tjenesten her:
    outbound:
      external:
        - host: <app>.<dev|prod>-fss-pub.nais.io
Se https://docs.nais.io/clusters/migrating-to-gcp/#how-do-i-reach-an-application-found-on-premises-from-my-application-in-gcp